source/geometry/FloatBox2D.ooc

```diff
@@ -40,8 +40,8 @@ FloatBox2D: cover {
 	rightCenter ::= FloatPoint2D new(this right, this center y)
 	topCenter ::= FloatPoint2D new(this center x, this top)
 	bottomCenter ::= FloatPoint2D new(this center x, this bottom)
-	empty ::= this size empty
-	area ::= this size x * this size y
+	isEmpty ::= this size hasZeroArea
```

Comment:
Ah, got it. In our 3D world, it's quite hard to store stuff in a "large" 2D box anyhow. :) 
But I see the problem. I'm OK with `hasZeroArea` for box as well. What do you think?